src/main.rs

```diff
@@ -140,6 +140,8 @@ pub struct GitHubReleaseAddOpts {
 
 impl GitHubReleaseAddOpts {
     pub fn add(&self) -> Result<(String, Pin)> {
+        log::warn!("The releases API always gives you the *latest* release, which is probably not what you want!");
```

Comment:
Arguably this is the right choice in *many* situations. I am not against this warning but perhaps the wording might need improvement?